app/views/casa_cases/_form.html.erb

```diff
@@ -20,10 +20,13 @@
         </div>
       <% end %>
 
-      <div class="field form-group">
-        <%= form.label :transition_aged_youth %>
-        <%= form.check_box :transition_aged_youth %>
-      </div>
+      <% if !casa_case.persisted? %>
```

Comment:
I think it's easier to not read negatives (despite the structure of this sentence) 😬 :

```
<% if casa_case.new_record? %>
```

Reply:
You are absolutely right. Changed it!